include/mc_tasks/lipm_stabilizer/StabilizerTask.h

```diff
@@ -282,14 +284,36 @@ struct MC_TASKS_DLLAPI StabilizerTask : public MetaTask
    * @param comd Desired CoM velocity
    * @param comdd Desired CoM acceleration
    * @param zmp Desired ZMP
+   * @param zmpd Desired ZMP velocity (can be omitted when zmpdGain in StabilizerConfiguration is zero)
    *
    * \see staticTarget for a helper to define the stabilizer target when the CoM
    * is static
    */
   void target(const Eigen::Vector3d & com,
               const Eigen::Vector3d & comd,
               const Eigen::Vector3d & comdd,
-              const Eigen::Vector3d & zmp);
+              const Eigen::Vector3d & zmp,
+              const Eigen::Vector3d & zmpd = Eigen::Vector3d::Zero());
+
+  /**
+   * @brief Set the wrench that the robot expects to receive from the external contacts.
+   *
+   * @param extWrenches External wrenches, which is represented by the vector of the pair of surface name and wrench in
+   * the surface frame
+   */
+  void setExtWrenches(const std::vector<std::pair<std::string, sva::ForceVecd>> & extWrenches);
```

Comment:
I'd prefer `setExternalWrenches`

Reply:
Fixed in https://github.com/jrl-umi3218/mc_rtc/pull/95/commits/3cebb45ceab415c3d5c6f302494e6f8254318890